libs/community/langchain_community/llms/mlx_pipeline.py

```diff
@@ -135,9 +135,32 @@ def _call(
                 "Please install it with `pip install mlx_lm`."
             )
 
-        pipeline_kwargs = kwargs.get("pipeline_kwargs", {})
+        pipeline_kwargs = kwargs.get("pipeline_kwargs", self.pipeline_kwargs)
 
-        return generate(self.model, self.tokenizer, prompt=prompt, **pipeline_kwargs)
+        temp: float = pipeline_kwargs.get("temp", 0.0)
+        max_tokens: int = pipeline_kwargs.get("max_tokens", 100)
```

Comment:
100 seems very small — is that number coming from somewhere? does max_tokens need to be specified to generate at all, or would it be better to leave it as None if user doesn't set it?

Reply:
Hey @baskaryan 

Thanks for the quick review!

100 is small so the generation can be quick and is also the default value also used in [MLX-LM](https://github.com/Blaizzy/mlx-examples/blob/main/llms/mlx_lm/utils.py#L201C1-L212C10). 

If the user wants they can pass a bigger number this ways:
```
from langchain_community.llms.mlx_pipeline import MLXPipeline

pipe = MLXPipeline.from_model_id(
    "mlx-community/dolphin-2.8-mistral-7b-v02-4bit",
    pipeline_kwargs={"max_tokens": 5000},
)
```